src/shared/components/horizontal-scroller/arrows/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
```

Comment:
This component renders a single Arrow. I would call the file simply `arrow` or `arrow-button`